src/feature.js

```diff
@@ -44,6 +44,10 @@ var feature = function (arg) {
       m_dependentFeatures = [],
       m_selectedFeatures = [];
 
+  // subclasses can add keys to this for styles that apply to subcomponents of
+  // data items, such as individual vertices on lines or polygons.
+  this._subcomponentStyles = {};
```

Comment:
should we just call it componentStyles? or perFeatureStyle? sub and component in my understanding convey very similar message.